index.js

```diff
@@ -21,6 +21,41 @@ app.get('/api/v1/favorites', (request, response) => {
     });
 });
 
+app.get('/api/v1/favorites/:id', (request, response) => {
+  database('favorites').where('id', request.params.id).select()
+    .then((favorites) => {
```

Comment:
Do you have reasoning for this to be favorites instead of favorite?

Reply:
nope not at all, i'll change that real quick.